sql/2021/performance/LCP_element.sql

```diff
@@ -0,0 +1,13 @@
+#standardSQL
+# LCP element node
+
+SELECT
+  _TABLE_SUFFIX AS client,
+  JSON_EXTRACT(payload, "$._performance.lcp_elem_stats[0].nodeName") AS lcp_node,
+  COUNT(DISTINCT url) AS total_urls,
```

Comment:
The stats should all be relative to the size of the sample set. For example, it's unclear if 4227 pages having an LCP node of IMG is high or low. If we divide that by the total number of pages, it's a lot clearer:

```suggestion
  COUNT(DISTINCT url) AS pages,
  SUM(COUNT(DISTINCT url)) OVER (PARTITION BY _TABLE_SUFFIX) AS total,
  COUNT(DISTINCT url) / SUM(COUNT(DISTINCT url)) OVER (PARTITION BY _TABLE_SUFFIX) AS pct
```

The `pct` field shows that in this example the adoption is 47%, which is easier to reason about and comparable across annual datasets that change in size.

Reply:
Cool, I assumed I could just do percentages and whatnot in Google Sheets and keep the queries simpler.